doc/example/conversion_reaction_conditions_multiple/model_conversion_reaction.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I was wondering too. If it is the base model extended for multiple conditions, it would be fine, but in general it would be nice to keep the number of models included low, so if there are synergies, preferable.

Reply:
Moved into a subfolder: `doc/example/conversion_reaction/multiple_conditions/`, with all identical files reused.